systemtest/infra.go

```diff
@@ -116,6 +116,8 @@ func SuffixTopics(topics ...apmqueue.Topic) []apmqueue.Topic {
 		suffixed[i] = apmqueue.Topic(
 			fmt.Sprintf("%s.%s", strings.ToLower(string(topics[i])), suffix),
 		)
+		suffixed[i] = apmqueue.Topic(strings.ReplaceAll(string(suffixed[i]), "_", "-"))
+		suffixed[i] = apmqueue.Topic(strings.ReplaceAll(string(suffixed[i]), "/", "-"))
```

Comment:
I wonder if it would be clearer to do theses replace against strings, and turn them into `apmqueue.Topic` only once.
Maybe moving it to a utility method?

Reply:
Done!